enigma-js/test/Enigma.spec.js

```diff
@@ -1,14 +1,24 @@
 /* eslint-disable require-jsdoc */
 import dotenv from 'dotenv';
+import fs from 'fs';
+import os from 'os';
+import path from 'path';
 import Enigma from '../src/Enigma';
 import utils from '../src/enigma-utils';
 import forge from 'node-forge';
 import Web3 from 'web3';
 import JSBI from 'jsbi';
 import EthCrypto from 'eth-crypto';
 import EnigmaContract from '../../build/contracts/Enigma';
+import EnigmaContractV2 from '../../build/contracts/EnigmaV2';
 import EnigmaContractSimulation from '../../build/contracts/EnigmaSimulation';
+import EnigmaContractSimulationV2 from '../../build/contracts/EnigmaSimulationV2';
 import EnigmaTokenContract from '../../build/contracts/EnigmaToken';
+import WorkersImplContract from '../../build/contracts/WorkersImpl';
```

Comment:
Looks like this is not needed, you don't use it anywhere.

Reply:
Resolved in a6843dc838e74a9c47cbcbc5093ad66c8e702dd3